src/main/index.js

```diff
@@ -26,11 +26,25 @@ import packageDetails from '../../package.json'
 const brotliDecompressAsync = promisify(brotliDecompress)
 
 if (process.argv.includes('--version')) {
+  console.log(`v${app.getVersion()}`) // eslint-disable-line no-console
```

Comment:
To avoid confusion about version mismatches, it is probably best to get it as close to the one shown on the About page as possible (it won't be translated like on the about page but at least it will have the beta text).
```suggestion
  console.log(`v${packageDetails.version} Beta`) // eslint-disable-line no-console
```